ProPublicaCongressAPI/InternalModels/SpecificBill.cs

```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
         [JsonProperty("sponsor")]
         public string SponsorMemberName { get; set; }
 
+        [JsonProperty("sponsor_id")]
+        public string SponsorId { get; set; }
+
         [JsonProperty("sponsor_uri")]
         public string SponsorMemberDetailUrl { get; set; }
 
+        [JsonProperty("sponsor_party")]
+        public string SponsorParty { get; set; }
+
+        [JsonProperty("sponsor_state")]
+        public string SponsorState { get; set; }
+
         [JsonProperty("gpo_pdf_uri")]
         public string BillDocumentPdfUrl { get; set; }
 
+        [JsonProperty("congressdotgov_url")]
+        public string CongressDotGovUrl { get; set; }
+
+        [JsonProperty("govtrack_url")]
+        public string GovTrackUrl { get; set; }
+
         [JsonProperty("introduced_date")]
         public string DateIntroduced { get; set; }
 
+        [JsonProperty("active")]
+        public bool Active { get; set; }
+
+        [JsonProperty("house_passage")]
+        public string HousePassage { get; set; }
+
+        [JsonProperty("senate_passage")]
+        public string SenatePassage { get; set; }
+
+        [JsonProperty("enacted")]
+        public string Enacted { get; set; }
+
+        [JsonProperty("vetoed")]
```

Comment:
Any idea if this is a true/false like the `active` property? The documentation just has blanks.